cognite/neat/_rules/transformers/_mapping.py

```diff
@@ -208,3 +211,128 @@ def _find_overwrites(self, prop: DMSProperty, mapping_prop: DMSProperty) -> tupl
                     # These are used for warnings so we use the alias to make it more readable for the user
                     conflicts.append(mapping_prop.model_fields[field_name].alias or field_name)
         return to_overwrite, conflicts
+
+
+class AsParentName(RulesTransformer[DMSRules, DMSRules]):
```

Comment:
We are not changing property name but property id, so suggest calling this 
`AsParentPropertyId` as that is what we are doing here